src/Uno.UI/UI/Xaml/Controls/ContentControl/ImplicitTextBlock.cs

```diff
@@ -15,7 +15,11 @@ namespace Windows.UI.Xaml.Controls
 	[Data.Bindable]
 	public partial class ImplicitTextBlock : TextBlock
 	{
-		public ImplicitTextBlock() { }
+		public ImplicitTextBlock(DependencyObject parent)
```

Comment:
Is the reference to the ContentControl being held? Ie is there any risk of a memory leak?

Reply:
No. It only needs its parent to retrieve its `AutomationProperties.AccessibilityView` value once.